.eslintrc.json

```diff
@@ -21,7 +21,7 @@
       ],
       "extends": [
         "plugin:tailwindcss/recommended",
-        "airbnb-typescript",
+        "airbnb-typescript/base",
```

Comment:
[Question] Why change the path?

Reply:
was a suggested fix but its not an error anymore, removing it now